internal/k8s/volume_finder.go

```diff
@@ -75,7 +75,7 @@ func (f VolumeFinder) GetPersistentVolumes(ctx context.Context) ([]VolumeInfo, e
 	}
 
 	for _, volume := range volumes.Items {
-		if Contains(f.DriverNames, volume.Spec.CSI.Driver) {
+		if volume.Spec.CSI != nil && Contains(f.DriverNames, volume.Spec.CSI.Driver) {
```

Comment:
Should the code change be applied to other `volume_finder.go` in `csm-metrics-xxx`?

Reply:
@baoy1 I remember In csm-metrics-xxx, it has filter?

Reply:
That's true. Thanks for reminding.

Reply:
That's true. Thanks for reminding.